<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QDialog" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	border: 2px solid grey;
	border-radius: 10px;
	padding: 0 8px;
	background: grey;
   color:white;
}
QTextEdit
{
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background: White;
color:black;
}
QListWidget
{
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background: White;
color:black;
}
*
{
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -
0.4, radius: 1.35, stop: 0 #fff, stop: 1 #888);
color: rgb(0, 0, 0);
border: 1px solid #ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="textEditConfirmation">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Split Count :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCount"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>PassWord : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPassWd">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>splitter</zorder>
  <zorder>textEditConfirmation</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
